### PR TITLE
Host container update for v1.17.0

### PR DESCRIPTION
**Pull request by jpculp**
**Description of changes:**

Update the default admin and control container images to [v0.11.2](https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/89) and [v0.7.6](https://github.com/bottlerocket-os/bottlerocket-control-container/pull/52) respectively.

**Testing done:**

* [x] `cargo make check-migrations`
* [x] Verify upgrade/downgrade

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: Release.toml [hunk 1]
```diff
@@ -246,6 +246,10 @@ version = "1.16.1"
     "migrate_v1.16.0_public-control-container-v0-7-5.lz4",
     "migrate_v1.16.0_schnauzer-v2-generators.lz4",
 ]
```
Comment (etungsten) [inline Release.toml]: Missing 1.16.0 -> 1.16.1 chain.

**Review by etungsten — CHANGES_REQUESTED** [review]

**Review by cbgbt — COMMENTED** [review]
This seems right to me, assuming testing goes well.

File: sources/api/migration/migrations/v1.17.0/aws-admin-container-v0-11-2/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+use migration_helpers::common_migrations::ReplaceSchnauzerMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+const OLD_ADMIN_CTR_CMDLINE: &str =
+    "schnauzer-v2 render --requires 'aws@v1(helpers=[ecr-prefix])' --template '{{ ecr-prefix settings.aws.region }}/bottlerocket-admin:v0.11.1";
```
Comment (etungsten) [inline sources/api/migration/migrations/v1.17.0/aws-admin-container-v0-11-2/src/main.rs]: Missing closing `'` same below

File: sources/api/migration/migrations/v1.17.0/aws-control-container-v0-7-6/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+use migration_helpers::common_migrations::ReplaceSchnauzerMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+const OLD_CONTROL_CTR_CMDLINE: &str =
+    "schnauzer-v2 render --requires 'aws@v1(helpers=[ecr-prefix])' --template '{{ ecr-prefix settings.aws.region }}/bottlerocket-control:v0.7.5";
```
Comment (etungsten) [inline sources/api/migration/migrations/v1.17.0/aws-control-container-v0-7-6/src/main.rs]: Missing `'` and below

**Review by etungsten — CHANGES_REQUESTED** [review]

**Comment by etungsten** [comment]
Tested migrations and host-containers source image URLS are updated correctly:

Upgrade:
```
[ssm-user@control]$ apiclient get os
{
  "os": {
    "arch": "x86_64",
    "build_id": "763f6d4c",
    "pretty_name": "Bottlerocket OS 1.16.1 (aws-k8s-1.28)",
    "variant_id": "aws-k8s-1.28",
    "version_id": "1.16.1"
  }
}
...                 
bash-5.1# updog whats
aws-k8s-1.28 1.17.0
bash-5.1# updog update -r -n
Starting update to 1.17.0
Reboot scheduled for Thu 2023-12-07 21:19:18 UTC, use 'shutdown -c' to cancel.
Update applied: aws-k8s-1.28 1.17.0
bash-5.1# reboot
...
[root@admin]# apiclient get os                                                                          
{
  "os": {
    "arch": "x86_64",
    "build_id": "0157a90a",
    "pretty_name": "Bottlerocket OS 1.17.0 (aws-k8s-1.28)",
    "variant_id": "aws-k8s-1.28",
    "version_id": "1.17.0"
  }
}
[root@admin]# apiclient get settings.host-containers
{
  "settings": {
    "host-containers": {
      "admin": {
        "enabled": true,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-admin:v0.11.2",
        "superpowered": true,
        "user-data": "..."
      },
      "control": {
        "enabled": true,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-control:v0.7.6",
        "superpowered": false
      }
    }
  }
}


```

Downgrade:
```bash
bash-5.1# signpost rollback-to-inactive
bash-5.1# reboot
[ssm-user@control]$ apiclient get os
{
  "os": {
    "arch": "x86_64",
    "build_id": "763f6d4c",
    "pretty_name": "Bottlerocket OS 1.16.1 (aws-k8s-1.28)",
    "variant_id": "aws-k8s-1.28",
    "version_id": "1.16.1"
  }
}
[ssm-user@control]$ apiclient get settings.host-containers
{
  "settings": {
    "host-containers": {
      "admin": {
        "enabled": true,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-admin:v0.11.1",
        "superpowered": true,
        "user-data": "
      },
      "control": {
        "enabled": true,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-control:v0.7.5",
        "superpowered": false
      }
    }
  }
}


```

**Review by etungsten — APPROVED** [review]

**Review by cbgbt — APPROVED** [review]

**Pull request merged**